datasources.tf

```diff
@@ -0,0 +1,3 @@
+data "openstack_networking_network_v2" "network" {
```

Comment:
I'm thinking it might be better if we prefixed these with ``keepalived`` so we can identify these better in case we need to manually clean them up. Ideally we should do this globally in other cookbooks.